xstate-chatbot/nodejs/src/machine/chat-machine.js

```diff
@@ -95,7 +95,7 @@ const chatStateMachine = Machine({
             },
             {
               cond: (context) => context.intention == 'rrt',
-              target: '#rrtSrfId'
+              target: '#rrtMobileNumber'
```

Comment:
From here, if we are going to "rrtMobileNumber", then is "rrtSrfId" being used anywhere? 
If it isn't being used, please remove it.

Reply:
rrtSrfId is not used as of now but in next RMO flow it is going to be used.thats why it is available as a state only but it is not there in chat-machine.js